src/cluster_legacy.c

```diff
@@ -5793,12 +5802,12 @@ int handleDebugClusterCommand(client *c) {
 
     /* Terminate the link based on the direction or all. */
     if (!strcasecmp(c->argv[3]->ptr, "from")) {
-        freeClusterLink(n->inbound_link);
+        if (n->inbound_link) freeClusterLink(n->inbound_link);
```

Comment:
Aren't these checks sufficient? rcvbuf and send_msg_queue are created and freed together. I suppose the other fix would be to do a NULL check at the beginning of freeClusterLink().